tensorflow/compiler/mlir/xla/transforms/hlo_legalize_to_lhlo.cc

```diff
@@ -316,10 +316,13 @@ class HloToLhloTensorStoreOpConverter : public ConversionPattern {
 //               %arg1: memref<4xf32>,
 //               %arg2: memref<4xf32>) {
 //   %0 = alloc() {temp = true} : memref<4xf32>
+//   %1 = alloc() {temp = true} : memref<4xf32>
 //   "xla_lhlo.max"(%arg0, %arg1, %0) {name = "maximum.47"} :
 //         (memref<4xf32>, memref<4xf32>, memref<4xf32>) -> ()
-//   "xla_lhlo.add"(%arg0, %0, %arg2) {name = "maximum.47"} :
+//   "xla_lhlo.add"(%arg0, %0, %1) {name = "maximum.47"} :
```

Comment:
nit: since you are already here, please remove `name = "stuff"` attributes from documentation

Reply:
Done.